### DISCUSSION TITLE
Removing obscure tags

### DISCUSSION THREAD

**Discussion by WGroleau** [nicfit/eyeD3#636] [General]
If I edit an MP3 containing a "Unique file ID" (UFID), obviously the tag is no longer valid.  How do I remove it (or any other tag not mentioned in the `--help`) ?
My workaround was to paste the audio into a new file and re-create all the useful tags.

**Comment by nicfit** [comment]
`--remove-frame FID    Remove all frames with the given ID. This option may be specified multiple times.`
